pkg/runner/cluster_k8s.go

```diff
@@ -219,6 +219,24 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow *rpc
 				}},
 		})
 
+		podCPU := defaultCPU
```

Comment:
just a style thing, but we have another way for deriving default configs in the docker runner. Not sure why we use mergo.Merge there and just if-statements in the k8s-runner.

Reply:
I will check how we do that in the local runner and see if we can align in a future PR.